test/e2e/scheduling/priorities.go

```diff
@@ -209,8 +222,6 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		// make the nodes have balanced cpu,mem usage ratio
 		err := createBalancedPodForNodes(f, cs, ns, nodeList.Items, podRequestedResource, 0.5)
 		framework.ExpectNoError(err)
-		//we need apply more taints on a node, because one match toleration only count 1
-		By("Trying to apply 10 taint on the nodes except first one.")
```

Comment:
Any particular reason for deleting this?

Reply:
Thanks for catching, you are right, I was thinking I was changing the test I am introducing as part of this PR. I was not intending to touch this.